src/components/node-editor/nodeFromSchema.ts

```diff
@@ -15,6 +15,8 @@ function defaultValue(io: InputOutput): unknown {
       return io.params.min ?? 0;
     case "Enum":
       return (io.params.items as unknown[])[0];
+    case "Color":
+      return "#95C624";
```

Comment:
Any reason why this color in particular?

Reply:
It's the green in the opensight logo